res/text/acuity_cavern.json

```
@@ -0,0 +1,12 @@
+{
+  "key": 9291,
+  "AcuityCavern_Text_UxieCry": {
+    "en_US": "Kyouuuun!\\r"
+  },
+  "AcuityCavern_Text_UxieDisappeared": {
+    "en_US": [
+      "UXIE disappeared deep into its\\n",
+      "cavern..."
+    ]
+  }
+}
```

```suggestion
{
  "key": 9291,
  "messages": [
    {
      "id": "AcuityCavern_Text_UxieCry",
      "message": "Kyouuuun!\\r"
    },
    {
      "id": "AcuityCavern_Text_UxieDisappeared",
      "message": [
        "UXIE disappeared deep into its\\n",
        "cavern..."
      ]
    }
  ]
}
```
The message ids cannot be used as keys since JSON does not preserve ordering of keys, so they have to be placed in an array.

---

It is implementation-dependent, as explained above. Some libraries will maintain key-ordering. Some will not. Python's native `json` library maintains key-ordering, as does RapidJSON for C++. Even `nlohmann/json` can be configured to maintain key-ordering.

---

Sorry, missed out on the earlier comment. But that's exactly the point - we should follow the spec instead of relying on implementations of individual libraries.

---

I can get behind spec-compliance as a general-purpose rule.

We'll need to do an audit of existing files to update them to be spec-compliant. I can make an issue for that as follow-up.